src/components/Modal/index.js

```diff
@@ -62,32 +82,53 @@ export default class Modal extends Component {
       children,
       className,
       HeaderComponent,
+      closeButtonPosition,
+      mountEntry,
       ...props
     } = this.props
+    const cnRoot = 'mc-components-modal'
 
     const classNames = cn(
-      'modal__container',
+      `${cnRoot}__container`,
```

Comment:
Correct me if I'm wrong, @stephenp , but this should just be the root block `"mc-components-modal"` (which would just be `cnRoot`.

Reply:
yeah, I think the class name should be something like `mc-modal` at the root, and then `mc-modal__container` inside if necessary